### PR TITLE
Add iOS Framework

### LINKED ISSUE #22: iOS framework

### PR DESCRIPTION
**Pull request by 335g**
Fix #22

### CODE REVIEW

File: Logician.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -282,6 +431,86 @@
...
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				DEFINES_MODULE = YES;
+				DYLIB_COMPATIBILITY_VERSION = 1;
+				DYLIB_CURRENT_VERSION = 1;
+				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				INFOPLIST_FILE = Sources/Info.plist;
+				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "jp.335g.Logician-iOS";
```
Comment (mdiep) [inline Logician.xcodeproj/project.pbxproj]: Would you mind changing this to match the bundle identifier of the macOS target? `com.diephouse.matt.Logician`?

File: Logician.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -282,6 +431,86 @@
...
+		84F4072B1E29B9EC00ADD734 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				INFOPLIST_FILE = Tests/LogicianTests/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.2;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "jp.335g.Logician-iOSTests";
```
Comment (mdiep) [inline Logician.xcodeproj/project.pbxproj]: Would you mind switching this to the match the bundle identifier of the mac tests? `com.diephouse.matt.LogicianTests`

File: Logician.xcodeproj/project.pbxproj (hunk 1 above)
Comment (335g) [inline Logician.xcodeproj/project.pbxproj]: Oh, I forgot it. 😓 Fixed 👍

File: Logician.xcodeproj/project.pbxproj (hunk 2 above)
Comment (335g) [inline Logician.xcodeproj/project.pbxproj]: Oh, I forgot it. 😓 Fixed 👍

**Comment by 335g** [comment]
Thanks for review 🙇 
I added iOS target to `.travis.yml` . I am not familiar with Travis, is this OK ?

**Comment by mdiep** [comment]
Yeah, that's fantastic! Thank you so much!

**Comment by 335g** [comment]
🙏 ✨

**Pull request merged**